charts/osm/values.yaml

```diff
@@ -16,6 +16,14 @@ OpenServiceMesh:
   imagePullSecrets: []
   # -- Envoy sidecar image
   sidecarImage: envoyproxy/envoy-alpine:v1.17.0
+  osmcontroller:
```

Comment:
Could you regenerate the chart README with `make chart-readme` and commit it as a part of this change if there is a diff generated.